src/style/content/preferences/devices.less

```diff
@@ -18,9 +18,29 @@
  */
 
 .preferences-devices-model {
-  font-size: @font-size-base;
+  display: flex;
+  align-items: center;
+  font-size: var(--font-size-base);
+  font-weight: var(--font-weight-semibold);
+  line-height: var(--line-height-lg);
+
+  .conversation-badges {
+    margin-left: 4px;
+  }
+}
+
+.preferences-devices-model-name {
+  display: flex;
+  align-items: center;
+
+  margin-bottom: 32px;
+  font-size: @font-size-large;
   font-weight: @font-weight-medium;
   line-height: @line-height-lg;
```

Comment:
suggestion: shall we use the css var here? like you did here https://github.com/wireapp/wire-webapp/pull/15637/files#diff-276c9caaec4603828380a2576916de8e6daa3550517ff22debcf4e0453030ed1R21-R25